### ISSUE TITLE
Feature request: write debug logs by default.

### ISSUE THREAD

**Issue by 5chdn** [ethereum/go-ethereum#2495]
#### System information

Geth version: `any`
OS & Version: Windows/Linux/OSX
Commit hash : `any`

#### Expected behaviour

Write a log file with usefull logs on INFO or DEBUG level by default.

As a moderator on Stack Exchange, I'm at the frontline of user support and I keep getting questions like [why is my connected peers so few and blocks syncronising so slow even it stops](http://ethereum.stackexchange.com/questions/3355/87).

My first reaction is to ask about which version they are running and which backend. Sometimes they don't even know it's geth. And also not which version.

The second reaction would be to ask what appears in the logs. But since most users simply double click the mist wallet dapp, the logs are lost in stdout somewhere.

#### Actual behaviour

User has to manually pipe the log to a file (and to know how to use a command line).

#### Steps to reproduce the behaviour

Run `geth`.

#### Rationale

I fear this might be closed as "works as desired", I see [piping is the way to go](https://github.com/ethereum/go-ethereum/wiki/Getting-Geth#using-stderr). But at least I would like to get an opinion, why there is no debug logging enabled and maybe an indication if this could be implemented or not.

**Comment by fjl** [comment]
Thank you for helping with user support!

Mist writes logs to a file by default. The log files are rotated and stored in a platform-specific directory.
On OS X, the latest log is `~/Library/Application Support/Mist/node.log`
On Windows, it is `C:\Users\<name>\AppData\Roaming\Mist\node.log`.
There used to be a button in the Mist menu that opens this folder.

geth itself is intended to run in many environments and logging to a file isn't always feasible or the best way. While we won't add log file support to geth in the near future, I hope that the Mist logs will make your life easier.

**Comment by fjl** [comment]
Oops. Didn't mean to close just yet.

**Comment by wawrzek** [comment]
I agree that geth approach to logs is "very interesting". I find it's very strange that geth doesn't send logs to expected location by default (e.g. /var/log/geth.log for Linux/UNIX, not sure what's on Windows). 

In what kind of environment environment not having logs are desire? Are they majority of us cases? If they are 'edge' cases I suggest to flip logic. Enable logs by default, but add option to stop them when needed.
